browser-test/src/support/index.ts

```diff
@@ -180,6 +180,14 @@ export const seedCanonicalQuestions = async (page: Page) => {
   await page.click('#canonical-questions')
 }
 
+export const enableFeatureFlag = async (page: Page, flag: string) => {
+  await page.goto(BASE_URL + '/dev/feature/$flag/enable')
+}
+
+export const disableFeatureFlag = async (page: Page, flag: string) => {
+  await page.goto(BASE_URL + '/dev/feature/$flag/disable')
```

Comment:
I believe this won't actually interpolate the String. I believe the correct interpolation syntax uses a backtick and `${`:

```
`/dev/feature/${flag}/disable`
```

Reply:
fixed! thank you!